deploy/monai-deploy-express/README.md

```diff
@@ -230,9 +252,14 @@ In this section, we will download a DICOM dataset, upload it to Orthanc and then
 In the `liver-lung-seg.json` workflow definition, we combined the AI Lung MAP and the AI Liver MAP into one single workflow definition. With this workflow definition, the Workflow Manager would route the incoming data based on the routing rules defined.
 
    1. Download one or both of the [datasets](#running-a-monai-deploy-workflow) provided above
-   2. Upload the dataset as described in [Uploading Data](#upload-dicom-datasets)
-   3. Deploy the workflow definition to MONAI Deploy Workflow Manager:
-   4. 
+   2. Download the AI Liver Tumor & the AI Lung Segmentation MAPs
+      ```bash
+      docker pull ghcr.io/mmelqin/monai_ai_lung_seg_app:1.0
+      docker pull ghcr.io/mmelqin/monai_ai_livertumor_seg_app:1.0
+      ```
+   3. Upload the dataset as described in [Uploading Data](#upload-dicom-datasets)
```

Comment:
The test data (DICOM series) is already imbedded in the container image, and the script to pull it is simple enough and covered in the App SDK discussion topic. We should transition to use the embedded test data instead of having to maintain other shared file(s).
The images are published under my own GitHub CR account, making it easier for me to update. We should consider archive the ones tested with the MD Express version in the MONAI ghcr repo, in case I mess up the new version of the images.

Reply:
It's much easier to click on a link to download IMO than run a script to extract the images from a docker image. 😸 
But we can get more feedback from users.

Reply:
I am one of the users, wearing QA hat. It creates burden on the contributors to maintain them, especially when the publisher of MD Express is different from the container image contributor. I can go ahead and create an issue for it, in any case.

Reply:
🤣  you are the creator so you don't count!